routes/indexRouter.js

```diff
@@ -6,12 +6,25 @@ const router = express.Router();
 
 //  base route leads to login page
 router.get("/", (req, res, next) => {
-  res.status(200).render("login", { css: ["login.css"] });
+  res
+    .status(200)
+    .render("login", { css: ["login.css"], js: ["login_form_validation.js"] });
 });
 
 // login page route
 router.get("/login", (req, res, next) => {
-  res.status(200).render("login", { css: ["login.css"] });
+  res
+    .status(200)
+    .render("login", { css: ["login.css"], js: ["login_form_validation.js"] });
+});
+
+// handle login post request
+router.post("/login", (req, res, next) => {
+  const { username, password } = req.body;
+
+  // TODO: query database to check if user exist
```

Comment:
Is this what you were working on when you said "I’m gonna start working on the login and registration"? Will that be a separate PR?

Reply:
yeah! I started to dive into and realized it would be better for me to create a new branch for that feature